java/mykola-melnyk/src/com/zoolatech/lecture1/tasks/_7/Main.java

```diff
@@ -0,0 +1,27 @@
+package com.zoolatech.lecture1.tasks._7;
+
+/**
+ * Write a program that prints all odd numbers from N to 1.
+ * N should be read from the input. Every number should be printed on the new line.
+ * Create two versions of the program: one should use the for loop, second - the while loop.
+ */
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Input an integer:");
+        int num = scanner.nextInt();
+        if (num % 2 == 0) {
+            System.out.println("Converting your even number to odd...");
+            int oddnum = num - 1;
+            for (int i = oddnum; i > 0; i -= 2)
```

Comment:
Right now you have two loops: one inside of the if block, second - inside of the else block. Could you please try to rewrite the if-else block in a such way that the program has only a single loop?

Reply:
Resolved: Removed one loop.